cms/migrations/0012_product_description_help_text.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.1.12 on 2021-09-17 12:24
+
+from django.db import migrations
+import wagtail.core.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cms', '0011_product_faculty_members_optional'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='coursepage',
+            name='description',
+            field=wagtail.core.fields.RichTextField(blank=True, help_text='The description shown on the home page and product page.'),
```

Comment:
Just run the `black` command so that it could follow pep8 line length restrictions.